sys-process/bottom/bottom-0.8.0.ebuild

```diff
@@ -238,6 +238,8 @@ src_prepare() {
 src_configure() {
 	myfeatures=(
 		$(usev battery)
+		$(usev zfs)
+		$(usev gpu)
```

Comment:
I would sort these alphabetically too, but some maintainers follows the order they appear in upstream's build system.